src/selectors.cc

```diff
@@ -420,6 +420,15 @@ select_to_reverse(const Context& context, const Selection& selection,
                   Codepoint c, int count, bool inclusive)
 {
     auto& buffer = context.buffer();
+
+    // if we are selecting backwards from the beginning of the buffer,
+    // there is nothing more that can be selected.
+    // Unless its inclusive, in which its possible that the current
+    // location of the cursor contains the Codepoint being looked for.
+    if(!inclusive && selection.cursor() == buffer.begin()){
```

Comment:
couple style issues:
- use `not` instead of `!`
- use `and` instead of `&&`
- add space around the if condition
- whe using braces, put them on a separate line

```suggestion
    if not inclusive and selection.cursor() == buffer.begin())
    {
```

also I'd recommend cleaning up the Git history. In this case you could run

	git reset $(git merge-base origin/master HEAD)

and then create a fresh commit